src/main/java/com/simibubi/create/content/redstone/displayLink/target/SignDisplayTarget.java

```diff
@@ -16,26 +15,31 @@ public class SignDisplayTarget extends DisplayTarget {
 	@Override
 	public void acceptText(int line, List<MutableComponent> text, DisplayLinkContext context) {
 		BlockEntity be = context.getTargetBlockEntity();
-		if (!(be instanceof SignBlockEntity sign))
+		if (!(be instanceof SignBlockEntity sign)){
 			return;
+		}
 
 		boolean changed = false;
-		SignText signText = new SignText();
+		SignText signTextFront = ((SignBlockEntity) be).getFrontText();
+		SignText signTextBack = ((SignBlockEntity) be).getBackText();
```

Comment:
Wouldn't that be better?

```suggestion
		SignText signTextFront = sign.getFrontText();
		SignText signTextBack = sign.getBackText();
```

Reply:
Also you don't need {} in every if statement, just in the `if (changed) { ... }`